Makefile

```diff
@@ -113,36 +113,44 @@ clean:
 ###                       Local testing using docker container              ###
 ###############################################################################
 # To start a 4-node cluster from scratch:
-# make clean && make build-linux && make localnet-start
+# make clean && make build-docker-node && docker-cluster-start
```

Comment:
nit - should the last one be `make docker-cluster-start`

Reply:
good catch, let me fix that now

Reply:
fixed